.github/workflows/publish-release.yml

```diff
@@ -25,7 +25,7 @@ jobs:
                     git config user.name ${{ secrets.BALLERINA_BOT_USERNAME }}
                     git config user.email ${{ secrets.BALLERINA_BOT_EMAIL }}
                     git checkout -b release-${VERSION}
-                    sed -i 's/ballerinaLangVersion=\(.*\)-SNAPSHOT/ballerinaLangVersion=\1/g' gradle.properties
+                    sed -i 's/ballerinaLangVersion=\(.*\)-[0-9]\{8\}-[0-9]\{6\}-.*$/ballerinaLangVersion=\1/g' gradle.properties
```

Comment:
Shall we keep both sed commands for SNAPSHOT as well as timestamp?

Reply:
+1